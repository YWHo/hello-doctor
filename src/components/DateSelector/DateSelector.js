import React from 'react';
import styled from 'styled-components';
import DaysBar from '../DaysBar';
import PartsOfDayBar from '../PartsOfDayBar';

const Container = styled.div`
  background-color: #3cb9c0;
  background-image: linear-gradient(to right, #3cb9c0, #2caaca);
  height: 148px;
`;

export default function DateSelector(props) {
  return (
    <Container>
      <DaysBar />
      <PartsOfDayBar />
    </Container>
  );
}
