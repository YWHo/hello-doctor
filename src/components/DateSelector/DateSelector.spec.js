import React from 'react';
import { shallow } from 'enzyme';
import DateSelector from './DateSelector';

describe('DateSelector', () => {
  it('render default correctly, ', () => {
    const wrapper = shallow(<DateSelector />);
    expect(wrapper).toMatchSnapshot();
  });
});
