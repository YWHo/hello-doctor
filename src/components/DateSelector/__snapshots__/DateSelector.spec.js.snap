// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateSelector render default correctly,  1`] = `
<styled.div
  $show={true}
>
  <Connect(DaySlots) />
  <Connect(PartsOfDayBar) />
</styled.div>
`;
